posttest.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('./test/init.js');
+
+var async = require('async');
+var _ = require('lodash');
+
+var db = global.getDataSource();
+var schemaName = global.config.schema;
+var findTables = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'T\'' + ';';
+var findViews = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'V\'' + ';';
+
+if (!process.env.CI)
+  return console.log('Skipping post-test ...');
+console.log('> Cleaning up the database ...');
+
+// drop all the tables and views from the schema
```

Comment:
I don't think `async.series()` is needed here. There are no series of tasks rather a list of **views** and **tables** that needs to be dropped and `async.each` iterates through it to do it.